wicker/testing/storage.py

```diff
@@ -5,7 +5,12 @@
 
 import pyarrow.fs as pafs
 
-from wicker.core.storage import S3DataStorage
+from wicker.core.storage import FileSystemDataStorage, S3DataStorage
+
+
+class FakeFileSystemDataStorage(FileSystemDataStorage):
+    def __init__(self, tmpdir: str = "/tmp") -> None:
```

Comment:
@isaak-willett are you using the `tmpdir` parameter here and if not, can it be removed?

Reply:
You know what I'm not, I was going to and didn't end up need it. Thanks for the callout, I totally removed this useless class.